pol-core/pol/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ warning_suppression(${ex_name})
 enable_pch(${ex_name})
 use_curl(${ex_name})
 use_benchmark(${ex_name})
+use_boost(${ex_name})
```

Comment:
Clib adds a dependency to the (old) boost target, to guarantee that it got extracted. Now the boost target builds boost, so nothing from the core can get build in parallel. Clib should be changed to depend on the extract target which is enough for all other Pol libs.